### PR TITLE
Create shortlist django project and api app within /src/backend/server is ISSUE #78

### LINKED ISSUE #78: Github - Create `shortlist` django project and `api` app within `/src/backend/server`
Anyone that would like to take this on, we need to get the django project and backend app scaffold setup and committed to `develop` (just like how the homework was started)

@gcivil-nyu-org/team-2-inperson

### PR DESCRIPTION
**Pull request by jainsvaibhav**

### CODE REVIEW

File: src/backend/server/server/settings.py [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+
+# Build paths inside the project like this: os.path.join(BASE_DIR, ...)
+BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/2.2/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = 'm#c)juz-3ypoqyf1yf(lefzohnowe-nv&s2ivo(&oi(_avwi3&'
```
Comment (waltage) [inline src/backend/server/server/settings.py]: Let's chat about using a .env file (or AWS secret manager) for this. We should change this value and not have it committed outright

File: src/backend/server/server/settings.py [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+
+# SECURITY WARNING: don't run with debug turned on in production!
+DEBUG = True
+
+ALLOWED_HOSTS = []
```
Comment (waltage) [inline src/backend/server/server/settings.py]: For nginx in prod, and for local environment, let's add localhost and some AWS targets here. I'll ping you the list.

We also need to add CSRF targets in order to allow posts

File: src/backend/server/server/settings.py [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+]
+
+WSGI_APPLICATION = 'server.wsgi.application'
+
+
+# Database
+# https://docs.djangoproject.com/en/2.2/ref/settings/#databases
+
+DATABASES = {
+    'default': {
```
Comment (waltage) [inline src/backend/server/server/settings.py:85]: Set this up with Postgres? I'm creating our AWS RDS instance this morning

Comment (waltage) [reply src/backend/server/server/settings.py:85]: I pinged you the dynamic settings for this to be read from a .env; let me know what you think

**Review by waltage — DISMISSED** [review]
A couple comments/options for how to modify the settings.py in order to be a little more secure, but also to allow an external config `.env` file option for us to make deployment changes without needing to change the underlying source code here

File: src/backend/server/server/settings.py (hunk 1 above)
Comment (waltage) [reply src/backend/server/server/settings.py]: I resolved this... I'll generate a new key and post it in slack for use with the .env file

File: src/backend/server/server/settings.py (hunk 2 above)
Comment (waltage) [reply src/backend/server/server/settings.py]: I resolved both of these

File: src/backend/server/server/settings.py (hunk 3 above)
Comment (waltage) [reply src/backend/server/server/settings.py:85]: I resolved these

**Review by yalexublue — APPROVED** [review]
LGTM, lets add this ASAP

**Pull request merged**